### ISSUE TITLE
[req]: Electro-Torrent

### ISSUE THREAD

**Issue by JaneQ14** [Jackett/Jackett#13951]
### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Is there already a request for your tracker?

- [X] I have checked older issues, open and closed

### Type

Semi-Private

### Tracker details

```markdown
- Name: Electro-Torrent
- URL: http://electro-torrent.pl/
- Description: Electro-Torrent is a POLISH Semi-Private Torrent Tracker for MOVIES / TV / MUSIC / GENERAL
- Minimum Seeding Time and Minimum Ratio (if Private): 
- Additional Comments: just register account (Same as Devil-Torrents but other stuff), without API
```
https not supported

**Comment by garfield69** [comment]
v0.20.2814

**Comment by ilike2burnthing** [comment]
http://electro-torrent.pl/ - ERR_CONNECTION_TIMED_OUT

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
http://electro-torrent.pl/ Erreur 503 - Service temporairement indisponible ou en maintenance

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
this semi-private site has changed their login back from login.php to logowanie since my update yesterday ;-(
but now going back to logowanie, I cannot get it to work at all, even switching to the cookie method.
so someone else can sort this out cause I've had it up to here with this ever changing site.

**Comment by garfield69** [comment]
this is the bit that is breaking the logowanie page processing for the indexer login attempt
```
<script src="https://www.google.com/recaptcha/api.js?render=6Lcj-pYUAAAAABFcaR3N3EDvHzu1cO5s_NrxUy_Y"></script>
<script>
    function frmSubmit(){
        grecaptcha.ready(function() {
            grecaptcha.execute('6Lcj-pYUAAAAABFcaR3N3EDvHzu1cO5s_NrxUy_Y', {action: 'recovery'}).then(function(token) {
                document.getElementById("captcha-response").value = token;                
                document.getElementById("form_id_recovery").submit();
            });
        });
    }    
```
we need to load captcha-response with the value somehow.

**Comment by garfield69** [comment]
took a fresh look. got it working with cookie + UA

**Comment by garfield69** [comment]
v0.21.258

**Comment by garfield69** [comment]
https://electro-torrent.pl/ Service temporarily unavailable

**Comment by ilike2burnthing** [comment]
back up, http only

**Comment by lukjod** [comment]
Some problems recently using indexer. It gives such info:
Jackett.Common.ExceptionWithConfigData: Login failed: Selector "a[href$="/logout.php"]" didn't match

**Comment by lukjod** [comment]
it works now, thx all is good

**Comment by garfield69** [comment]
https://electro-torrent.pl/ HTTP 500

**Comment by ilike2burnthing** [comment]
back up, just down for Beijing over on Uptrends

**Comment by garfield69** [comment]
while uptime is showing active I still get blank screen on browser with a http 500 on the devtools on both chrome and firefox and my jackett indexer shows `Exception (electro-torrent): Login failed: Selector "a[href$="/logout.php"]" didn't match: Parse error` but that could be because my cookie needs refreshing so catch 22
indexer working for you?

**Comment by ilike2burnthing** [comment]
Working in Jackett as well. Reconfigured with a new cookie and that works too. Want to mark as down anyway so you can monitor it?

**Comment by garfield69** [comment]
nah, its clearly just me and I'll test it again tomorrow when I perform my usual routine.

**Comment by garfield69** [comment]
I can use the browser to login, but when I use the cookie and UA to config indexer I get `Login failed: Selector "a[href$="/logout.php"]" didn't match`
Just me?

**Comment by garfield69** [comment]
now it is working %-/

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/13951#issuecomment-1445237628

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/13951#issuecomment-1445237628

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/13951#issuecomment-1747401934

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/13951#issuecomment-1675273268

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://electro-torrent.pl/ 522 Connection Timed Out

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/13951#issuecomment-2265907902

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://electro-torrent.pl/ > cloudflare  Connection timed out Error code 522

**Comment by garfield69** [comment]
online
